.github/workflows/go.yaml

```diff
@@ -3,7 +3,7 @@ name: go
 on:
   push:
     branches:
-      - main
+      - master
```

Comment:
Haven't we just renamed `master` to `main`? I guess this change shouldn't be here.

Reply:
good catch! fixed.